tests/integ/modin/resample/test_resample.py

```diff
@@ -477,3 +477,30 @@ def test_resample_ffill_large_gaps(interval):
         lambda df: df.resample(rule=f"{interval}D").ffill(),
         check_freq=False,
     )
+
+
+@freq
+@interval
+@sql_count_checker(query_count=3, join_count=1)
```

Comment:
why 3 queries? 2 from resample?

Reply:
All the tests in this file have 3 queries, 1 to convert index to native pandas for dataframe constructor, then 2 from the resample. Resample executes an eager query to get the min and max values of the index.

Below outside of `eval_snowpark_pandas_result` is only 2 queries:
```
    with SqlCounter(query_count=2):
        import modin.pandas as pd
        df = pd.DataFrame({"A": np.random.randn(15)}, index=native_pd.date_range("2020-01-01", periods=15, freq="1s"))
        df.asfreq(freq="5s", method="ffill").to_pandas()
```